system-tests/src/test/java/org/eclipse/edc/samples/transfer/Transfer06ConsumerPullHttpTest.java

```diff
@@ -0,0 +1,110 @@
+package org.eclipse.edc.samples.transfer;
+
+import org.eclipse.edc.junit.annotations.EndToEndTest;
+import org.eclipse.edc.junit.extensions.EdcRuntimeExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.time.Duration;
+import java.util.Map;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.After;
+import org.junit.Before;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.eclipse.edc.samples.transfer.FileTransferSampleTestCommon.getFileFromRelativePath;
+
+@EndToEndTest
+public class Transfer06ConsumerPullHttpTest {
+
+    static final String CONSUMER_CONFIG_PROPERTIES_FILE_PATH = "transfer/transfer-06-consumer-pull-http/http-pull-consumer/consumer-configuration.properties";
+
+    static final String PROVIDER_CONFIG_PROPERTIES_FILE_PATH = "transfer/transfer-06-consumer-pull-http/http-pull-provider/provider-configuration.properties";
+    static final String SAMPLE_ASSET_FILE_PATH = "transfer/transfer-06-consumer-pull-http/README.md";
+
+    private static String authCode="";
```

Comment:
Please add a space. 
```suggestion
    private static String authCode = "";
```
You may want to include the EDC codestyle files to your IDE settings. See [here](https://github.com/eclipse-edc/docs/blob/main/developer/contributing/styleguide.md) for a guide.